rubrics/services/gulpfile.js

```diff
@@ -0,0 +1,147 @@
+var gulp = require('gulp');
+var path = require("path");
+var uglify = require('gulp-uglify');
+var concat = require('gulp-concat');
+var rename = require('gulp-rename');
+var sass = require('gulp-sass');
+var stylemod = require('gulp-style-modules');
+var cleanCSS = require('gulp-clean-css');
+var clean = require('gulp-clean');
+var del = require('del');
+var vulcanize = require('gulp-vulcanize');
+var minifyInline = require('gulp-minify-inline');
+var bower = require('gulp-bower');
+var runSequence = require('run-sequence');
+
+// for development, change this to point to your tomcat installation 
+var server_base = "/opt/tomcat";
```

Comment:
This is the cause of the TravisCI failure. Diego is working on correcting this client code handling to separate the concerns of the web developer's direct use of Gulp during development vs how it is incorporated into the Maven build. It should be shifted to pass in the working target directory either via command line or local configuration (developer) or from the maven-frontend-plugin within a Maven build so that it uses the Maven ephemeral build directory (default target) for all client build tool activities and handle all interim processing such as downloading client dependencies for inclusion, such as the bower_components, and then overlaying the customization to the polymer-sortable.js file. That way everything is processed and packaged into the war artifact. The server_base should only come into play for a web developer who wants to code and refresh the running deployment during rapid development cycles without requiring any server side changes to be built and deployed.